src/components/FeedbackWidget.tsx

```diff
@@ -197,22 +203,22 @@ const FeedbackWidget = ({ className }) => {
     trackCustomEvent({
       eventCategory: `FeedbackWidget toggled`,
       eventAction: `Clicked`,
-      label: `Closed feedback widget`,
+      eventName: `Closed feedback widget`,
     })
   }
   const handleOpen = () => {
     setIsOpen(true)
     trackCustomEvent({
       eventCategory: `FeedbackWidget toggled`,
       eventAction: `Clicked`,
-      label: `Opened feedback widget`,
+      eventName: `Opened feedback widget`,
     })
   }
   const handleSubmit = (choice) => {
     trackCustomEvent({
       eventCategory: `Page is helpful feedback`,
       eventAction: `Clicked`,
-      label: choice,
+      eventName: choice,
```

Comment:
@wackerow do you know what should be the expected behavior here?

2 main concerns:
1. We were using `label` here, but it doesn't exist in the trackCustomEvent function.
2. If we want to use `eventName`, we should send a string but `choice` is a boolean. Should we send the stringified version of it, `"true" or "false"`?